keyboards/matrix/pjf/pjf.h

```diff
@@ -0,0 +1,23 @@
+/**
+ * pjf.h
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define KEYMAP_default( \
```

Comment:
This should be:
```suggestion
#define LAYOUT( \
```
here, and in the keymap.c.

Reply:
updated